portal-impl/src/com/liferay/portal/upgrade/v6_2_0/util/DLValidatorImpl.java

```diff
@@ -67,6 +69,11 @@ public long getMaxAllowableSize(long groupId, String mimeType) {
 		return 104857600;
 	}
 
+	@Override
+	public Map<String, Long> getMimeTypeSizeLimit(long groupId) {
+		return null;
```

Comment:
Can you return an empty map (`Collections.emptyMap()`) instead of null here?

Reply:
I updated!